components/TaskContainer.vue

```diff
@@ -1,108 +1,125 @@
-    >
-      <div class="">
         <div class="center-item">
-          <NoTaskLogo />
-        </div>
-        <div class="info-text margin-top-8 text-center">
-          {{ noTaskMessage }}
+          <button
+            v-if="loadMoreTask"
+            class="load-button text-button"
+            @click="loadMore"
+          >
+            {{ $t('load-more') }}
+          </button>
+          <button
+            v-if="showLessTask"
+            class="load-button text-button"
+            @click="showLess"
+          >
+            {{ $t('show-less') }}
+          </button>
         </div>
       </div>
-    </div>
-    <div
-      v-if="hasNoFilteredTask"
-      class="flex-grow-1 flex-box flex-direction-column center-item"
-    >
-      <div class="">
-        <div class="center-item">
-          <NoTaskLogo />
-        </div>
-        <div v-if="noCompletedTask" class="info-text margin-top-8 text-center">
-          {{ $t('NoCompleteTask') }}
+      <div
+        v-if="hasNoTask"
+        class="flex-grow-1 flex-box flex-direction-column center-item"
+      >
+        <div class="">
+          <div class="center-item">
+            <NoTaskLogo />
+          </div>
+          <div class="info-text margin-top-8 text-center">
+            {{ noTaskMessage }}
+          </div>
         </div>
-        <div v-if="noIncompleteTask" class="info-text margin-top-8 text-center">
-          {{ $t('NoIncompleteTask') }}
+      </div>
+      <div
+        v-if="hasNoFilteredTask"
+        class="flex-grow-1 flex-box flex-direction-column center-item"
+      >
+        <div class="">
+          <div class="center-item">
+            <NoTaskLogo />
+          </div>
+          <div
+            v-if="noCompletedTask"
+            class="info-text margin-top-8 text-center"
+          >
+            {{ $t('NoCompleteTask') }}
+          </div>
+          <div
+            v-if="noIncompleteTask"
+            class="info-text margin-top-8 text-center"
+          >
+            {{ $t('NoIncompleteTask') }}
+          </div>
         </div>
       </div>
     </div>
   </div>
 </template>
 <script>
 import { mapGetters, mapState } from 'vuex';
-import { LIMIT, SUCCESS, ERROR } from '../constants';
-import FilterComponent from '@/components/buttons/FilterComponent.vue';
-import DeleteIcon from '@/components/buttons/DeleteIcon.vue';
-import PlusIcon from '@/assets/svg/plusIcon.svg';
-import NoTaskLogo from '@/assets/svg/noTask.svg';
-import LoadingIcon from '@/components/buttons/LoadingIcon.vue';
-import global from '@/mixins/global';
+import { SUCCESS, ERROR } from '../constants';
+import PlusIcon from '../assets/svg/plusIcon.svg';
+import NoTaskLogo from '../assets/svg/noTask.svg';
+import LoadingIcon from '../components/buttons/LoadingIcon.vue';
+import global from '../mixins/global';
+import DeleteIcon from './buttons/DeleteIcon.vue';
+import FilterComponent from './buttons/FilterComponent.vue';
```

Comment:
ditto

Reply:
Adding relative path throughs error on test.

Reply:
We can investigate the problem. 
A good start [here](https://github.com/facebook/jest/issues/3499)

Reply:
resolved